src/codeu/chat/client/commandline/Chat.java

```diff
@@ -66,6 +96,19 @@ public boolean handleCommand(String line) {
       return true;
     }
 
+    if ("version-check".equals(command)) {
+      System.out.println("Version number: " + context.Version());
+      return true;
+    }
+
+    if ("up-time".equals(command)) {
+      Time serverTime = context.getTime();
+      Time now = Time.now();
```

Comment:
Computer time is a tricky subject that's often debated internally and the cause of many bugs.  I would recommend that the uptime be calculated server side and passed back rather than the start time passed and uptime calculated.  This removed the potential of bugs related to timezones and other computer time complexities.